### PR TITLE
Add Basalt->Blackstone recipe

### PR DESCRIPTION
**Pull request by svr333**
## Description
Adds a new recipe to convert 2 basalt into one blackstone (useful for nether only players)

## Proposed changes
New recipe to the grindstone

## Related Issues (if applicable)
[Discord suggestion 2007](https://discord.com/channels/565557184348422174/627048923122499584/993741632878940260)

## Checklist

- [ ] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [ ] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [ ] I have made sure that the proposed changes do not break compatibility across the supported Minecraft versions (1.14.* - 1.19.*).
- [x] I followed the existing code standards and didn't mess up the formatting.
- [ ] I did my best to add documentation to any public classes or methods I added.
- [ ] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values
- [ ] I added sufficient Unit Tests to cover my code.

### CODE REVIEW

File: src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/GrindStone.java [hunk 1]
```diff
@@ -78,6 +78,9 @@ protected void registerDefaultRecipes(@Nonnull List<ItemStack> recipes) {
         recipes.add(new ItemStack(Material.QUARTZ_BLOCK));
         recipes.add(new ItemStack(Material.QUARTZ, 4));
 
+        recipes.add(new ItemStack(Material.BASALT, 2));
+        recipes.add(new ItemStack(Material.BLACKSTONE));
```
Comment (TheBusyBiscuit) [inline src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/GrindStone.java:82]: Those are 1.16+ only items aren't they?

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/GrindStone.java (hunk 1 above)
Comment (svr333) [reply src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/GrindStone.java:82]: oh yeah, my bad

**Review by TheBusyBiscuit — DISMISSED** [review]
lgtm

File: src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/GrindStone.java [hunk 2]
```diff
@@ -78,6 +78,11 @@ protected void registerDefaultRecipes(@Nonnull List<ItemStack> recipes) {
         recipes.add(new ItemStack(Material.QUARTZ_BLOCK));
         recipes.add(new ItemStack(Material.QUARTZ, 4));
 
+        if (Slimefun.getMinecraftVersion().isAtLeast(MinecraftVersion.MINECRAFT_1_16)) {
```
Comment (WalshyDev) [inline src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/GrindStone.java:81]: Version check no longer required 😅 

```suggestion
        if (Slimefun.getMinecraftVersion().isAtLeast(MinecraftVersion.MINECRAFT_1_16)) {
```

**Review by Sefiraat — APPROVED** [review]
👍

**Review by Sfiguz7 — APPROVED** [review]
LGTM

**Review by TheBusyBiscuit — APPROVED** [review]

**Pull request merged**